app/scripts/modules/core/src/pipeline/config/actions/history/ShowPipelineHistoryModal.tsx

```diff
@@ -118,8 +118,8 @@ export function ShowPipelineHistoryModal(props: IShowHistoryModalProps) {
           )}
           {!loading && history.length > 1 && (
             <>
-              <div className="history-header row">
-                <div className="col-md-4">
+              <div className="history-header row horizontal">
+                <div className="revision-section col-md-4">
                   <FormField
                     label="Revision"
```

Comment:
this might be one of those cases where it would make sense to write an inline layout component instead of fight the css/markup in StandardFieldLayout (css which is likely going to change in the future). 

something like this might do the trick: 
```
layout={(label, input) => <div>{label}{input}</div>}
```

Reply:
you know, i originally thought 'aw man i'm gonna need my own layout', and then i had a spark of memory and said 'oh wait Chris overrode the min-width of this label in the Triggers component and it worked out great, i'm gonna steal that'. Guess I should've just tried out the "dumb" layout first.

Reply:
Turns out to get this looking right you actually need a fair amount of layout code (side-by-side flex with the right margin, bold text, force the input to grow to size), which makes the markup look _slightly_ closer to writing a standard layout:

```jsx
layout={({ label, input }) => (
  <div className="flex-container-h baseline margin-between-lg">
    <div className="bold">{label}</div>
    <div className="flex-grow">{input}</div>
  </div>
)}
```

I'm cool with/prefer that, but do you personally prefer having this level of layout smarts hard-coded inline versus needing to refactor CSS overrides later?

Reply:
The idea behind the field layouts is that there are a lot of things that look similar, and we can change them all in lockstep by updating the layout component.  I think overusing the layouts for things which _dont look similar_ will make it harder to iterate on the layout itself down the road.  I'd lean towards adding the markup.

In fact, `FormikFormField` components make it really easy to: reuse a standard layout, apply validation, handle state management.  `FormField` components make it easy to reuse a layout, and apply validation.  In this UI, the select widget doesn't really benefit from any of those things.  Maybe `FormField` isn't the right choice here at all here, and it should just be markup with a select inside it.

```js
 <div className="flex-container-h baseline margin-between-lg">
    <div className="bold">Revision</div>
    <div className="flex-grow"><SelectInput ... /></div>
  </div>
```

Reply:
>  'oh wait Chris overrode the min-width of this label in the Triggers component and it worked out great, i'm gonna steal that'

It's not clear to me if this is abusing the layout or not.  I suspect it's borderline abuse.

Reply:
yeah, it's not clear to me that the field components here are providing a ton of value. For now I'm gonna leave them as-is since this was mostly a quick dive in and out to fix this layout situation, but I'm certainly happier with the inline layout than coupling overrides to a field layout. Thanks for catching it 🙏